src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -91,13 +110,52 @@ const ProfileSelector = ({ input, showSelector }) => {
         isDisabled={!isSuccess}
       >
         {isSuccess &&
-          data.map((key, index) => <SelectOption key={index} value={key} />)}
+          [<SelectOption key={0} value={'None'} />].concat(
+            data.map((key, index) => (
+              <SelectOption key={index + 1} value={key} />
+            ))
+          )}
         {isFetching && (
           <SelectOption isNoResultsOption={true} data-testid="policies-loading">
             <Spinner size="md" />
           </SelectOption>
         )}
       </Select>
+      <TextContent>
+        <br />
+
+        <TextList component={TextListVariants.dl}>
+          <TextListItem
+            component={TextListItemVariants.dt}
+            className="pf-u-min-width"
+          >
+            Profile description:
+          </TextListItem>
+        </TextList>
+        <TextList component={TextListVariants.dl}>
+          <TextListItem
+            component={TextListItemVariants.dt}
+            className="pf-u-min-width"
+          >
+            Operating system:
+          </TextListItem>
+          <TextListItem
+            component={TextListItemVariants.dt}
```

Comment:
The TextLists work on a principal of term / description pair. So you'll have one `TextListItemVariants.dt` for the term and `TextListItemVariants.dd` for the description. Docs for this component are here: https://www.patternfly.org/components/text/#description-list

```suggestion
            component={TextListItemVariants.dd}
```

The `className="normal-text"` can also be removed as the `dd` formatting will keep the text not bold. :)